app/assets/javascripts/utils/request.js

```diff
@@ -15,5 +15,5 @@ export default (path, { method = 'GET', body = null, ...extraOptions } = {}) =>
   // If the path is an internal request, include the CSRF Token
   if (isRelativePath(path)) options.headers['X-CSRF-Token'] = Rails.csrfToken();
 
-  return fetch(path, body ? { body, ...options } : options);
+  return fetch(new URL(path, window.location.origin).href, body ? { body, ...options } : options);
```

Comment:
What warning(s) does this fix? The more explanation you can add in the commit messages, the easier things like this will be to evaluate.

Reply:
I'm sorry. The change in request.js solves the 'Only absolute URLs are supported' log error.